package.json

```diff
@@ -21,71 +21,79 @@
       "underscore": "lodash"
     }
   },
+  "ava": {
```

Comment:
I'd probably ditch AVA for a combination of Tape/Jasmine2/Chai + Karma. (Add Enzyme if testing React components.) See goblindegook/littlefoot for an example using Tape + Karma.